src/section.js

```diff
@@ -88,16 +88,9 @@ class Section {
 
 		this.load(_request).
 			then(function(contents){
-				var serializer;
-				var Serializer;
-
-				if (typeof XMLSerializer === "undefined") {
-					Serializer = require("xmldom").XMLSerializer;
-				} else {
-					Serializer = XMLSerializer;
```

Comment:
This uses the browsers XMLSerializer if it present.
Only need to load xmldom in node / worker environments.